docs/_renderer_core.py

```diff
@@ -30,6 +30,7 @@ class FileContentJson(TypedDict):
 
 class Renderer(MdRenderer):
     style = "shiny"
+    express_api = False
```

Comment:
This is what I'm doing in #1073 and would let us avoid having separate `_renderer` files for core and express.

```suggestion
    os.environ.get("SHINY_MODE") == "express"
```

Reply:
I'm going to merge #1073 into this branch hand then I'll handle this unless there are strong objections